docs/set_up_admin_tails.rst

```diff
@@ -136,18 +136,37 @@ signed with the release signing key:
 .. code:: sh
 
     cd ~/Persistent/securedrop/
-    git checkout 1.6.0
+    git fetch --tags
+    gpg --keyserver hkps://keys.openpgp.org --recv-key \
+    "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
     git tag -v 1.6.0
 
 You should see ``Good signature from "SecureDrop Release Signing Key"`` in the
-output of that last command along with the fingerprint above.
+output.
```

Comment:
This paragraph is duplicated by the new, improved language below, so you can remove it.